source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -46,9 +46,7 @@
 #include "mbedtls/cmac.h"
 #include "mbedtls/platform.h"
 #include "mbedtls/threading.h"
-
-/* Custom mbedtls utils include. */
-#include "mbedtls_error.h"
```

Comment:
I'm not sure how the CI checks are passing for this PR. The strerror function names changed between the freertos mbedtls_error.c implementation and the upstream version. 

mbedtls_strerror_highlevel -> mbedtls_high_level_strerr
mbedtls_srrerror_lowlevel -> mbedtls_low_level_strerr

Please refer to the upstream changes [over here](https://github.com/ARMmbed/mbedtls/blob/development/include/mbedtls/error.h).